migrations/2_DEV_local_development.js

```diff
@@ -21,9 +21,10 @@ module.exports = async function (deployer, network, accounts) {
 }
 
 async function deployTokens(deployer, owner) {
+  // TODO: get address from config/networks
   const GenToken = artifacts.require('GenToken') // GEN (Dao Stack)
   const MgnToken = artifacts.require('TokenFRT') // MGN (Token FRT)
```

Comment:
All the tokens but GEN are deployed in the `migrateDx` that was added in migration 2 of add-migration branch